lips/lip-9.md

```diff
@@ -0,0 +1,95 @@
+---
+lip: 9
+title: Libra Private Transaction Metadata
+authors: Daniel Prinz
+status: Draft
+type: Informational
+created: 09/10/2020
+---
+# Summary
+This LIP describes the conceptual model and implementation of a private, on-chain description attribute, that could be added to every P2P transaction. An example of that would be short and descriptive information about the transaction, like what are the purchased items that are being paid in a transaction. 
+
+# Abstract / Overview
+The Libra blockchain can carry informational metadata with every transaction made. This data could vary and usually used for validating both sides of the transaction could "understand" its mean. While this metadata deals with the functioning of the sender and receiver VASPs, and/or with the FATF travel rule regulatory constraints, the transaction info itself remains opaque to the end-users. This property is missing from the current payment experience of any traditional network.
+This LIP goal is to propose a standard way of describing any transaction while keeping the following principals:
+1. Indisputable data
```

Comment:
This property can also be satisfied by submitting a salted hash of the contents Vs the content itself (even if encrypted)

Reply:
Agree. 
The main reason for the data is the user control over it. If we go off-chain, a salted hash would be enough.